### DISCUSSION TITLE
Use neogit stage function outside of neogit window?

### DISCUSSION THREAD

**Discussion by molleweide** [NeogitOrg/neogit#259] [Q&A] [answered]
Hi there,

question:

Can one use a neogit function outside of neogit in a regular file, eg. such as the staging function in order to stage individual lines?
Or should I rather use the `gitsigns` plugin for this?

personal comments:

 I am kind of new to using git inside of vim so I am trying to understand how these git plugins relate to one another. I am now learning how to use `neogit`, `diffview`, `gitsigns` and I am sort of figuring that each of these are meant for their own specific purposes.

**Comment by TimUntersberger** [comment, marked as answer]
> Can one use a neogit function outside of neogit in a regular file, eg. such as the staging function in order to stage individual lines?

This is not supported for now.

**Reply by TimUntersberger** [reply]
I guess using gitsigns would be the correct choice.

**Reply by molleweide** [reply]
Allright, thank you very much for answering Tim!
